utils.mjs

```diff
@@ -41,6 +42,15 @@ const isPlainObject = (obj) => {
   return Object.prototype.toString.call(obj) === "[object Object]";
 };
 
+const allFirstNames = firstNames.male.concat(firstNames.female);
```

Comment:
spread대신 concat을 사용한 이유가 있나요?

Reply:
솔직하게 막연히 concat이 성능이 더 좋을 것이라고 생각해서 사용했는데 조사를 좀 해봤습니다.

일단 spread를 사용했을 때는 데이터의 크기가 너무 큰 경우 Maximum Callstack Exceeded 오류가 발생할 수 있다고 합니다. 물론 10만 건 이상의 데이터가 있을 때만 발생하는 것 같지만 그래도 많은 데이터를 다루게 될 것이라고 예상되므로 concat이 나쁘지 않은 선택이라고 생각됩니다. ([참고](https://www.educative.io/answers/spread-operator-vs-arrayprototypeconcat-in-javascript))

그 다음으로는 성능 측면에 장점이 있는데 단순하게 설명하자면 concat은 array에 특화된 매서드인 반면 spread는 이터러블 프로토콜에 부합하는 객체를 모두 인자로 받을 수 있는 매서드이기 때문에 최적화 기법이 다르다고 하더군요. ([출처](https://stackoverflow.com/questions/48865710/spread-operator-vs-array-concat))
> Performance-wise concat is faster, probably because it can benefit from array-specific optimizations, while ... has to conform to the common iteration protocol.

실제로 성능 테스트 결과도 있었습니다. ([출처](https://kir93.tistory.com/entry/Javascript-concat-vs-spread-vs-push-%EC%84%B1%EB%8A%A5%EC%97%90-%EB%8C%80%ED%95%B4))
![image](https://user-images.githubusercontent.com/43166681/194752950-1493a1eb-0a23-4f39-bd07-88c21210e72b.png)

Reply:
wow! 역시 블링 👍